src/APIFunctions/points.js

```diff
@@ -0,0 +1,90 @@
+const { request, gql } = require('graphql-request');
+const { DISCORD_API_URL } = require('../../config.json');
+const { ApiResponse } = require('./ApiResponses');
+
+/**
+ * @typedef {Object} Point
+ *
+ * @property {String} guildID The Discord server's ID.
+ * @property {String} userID The user's Discord ID.
+ * @property {Number} totalPoints Number of points gained overall.
+ * @property {Number} weekPoints Number of points gained since 7 days.
+ * @property {Number} monthPoints Number of points gained since 1st of month.
+ * @property {Number} yearPoints Number of points gained since Jan 1.
+ * @property {Date} lastTalked Date the user's last message was sent.
+ */
+
+// Function to query the user's points based on server ID and Discord ID
```

Comment:
```suggestion
/**
 * Query points for a user or users.
 *
 * @param args.guildID { String }
 * @param args.userID { String? } Optional. Query one user if specified, otherwise query entire guild
 * @returns { ApiResponse } Response with an array of matched user points.
 */
```

Please make JSDoc comments for the exported functions.